### PR TITLE
Add share option for blog posts

### PR DESCRIPTION
**Pull request by christian-bromann**
To allow the reader to easily share the content I added a section with various of options. I am using sharethis.com as simple integration. Seems to be easy and free and allows to analyse share behavior.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -60,7 +60,7 @@ google_analitycs_id = "" # Your ID
 # copyright
 copyright = "&copy; 2020 Sauce Labs Inc., all rights reserved. SAUCE and SAUCE LABS are registered trademarks owned by Sauce Labs Inc. in the United States, EU, and may be registered in other jurisdictions."
 # social links on hero
-enable_social_links_hero = false
+enable_social_links_hero = true
```
Comment (diemol) [inline config.toml]: Why is this needed?
This is different from what the designer recommended. Now we have a big red gap between the hero image and the first section.

**Review by diemol — CHANGES_REQUESTED** [review]

File: config.toml (hunk 1 above)
Comment (christian-bromann) [reply config.toml]: @diemol fixed

**Review by diemol — APPROVED** [review]
